twiki_wg/ssawg_trending_scraper.py

```diff
@@ -114,8 +114,9 @@ def parse_page(self):
         self.soup = BeautifulSoup(self.url_text, "lxml")
         if self.page != "celmon":
             for local_link in self.soup.find_all("a"):
-                temp = local_link["href"]
-                local_link["href"] = self.url + temp
+                if "href" in local_link:
+                    temp = local_link["href"]
+                    local_link["href"] = self.url + temp
```

Comment:
This is pointing to a mistake in `aimpoint_mon`. In https://github.com/sot/aimpoint_mon/commit/f3e5e2c07b947d3122de3384d9444fa962c7b524 the anchor tag should be done with:
```
<h3 id="aimpoint-trending"> Observed aimpoint differences trend </h3>
```

Reply:
Using `name` in an `<a>` tag is deprecated and doesn't exist in HTML5. (I didn't know that of course but that's what google says).

Reply:
I figured we probably had plenty of other "a" without hrefs as possible candidates for this local link code but OK.  And I'd figure if we move to using anchors with id that needs new logic here doesn't it?

Reply:
> I figured we probably had plenty of other "a" without hrefs as possible candidates for this local link code
A search of the `sot` org for `<a name` shows a number of matches but only aimpoint_mon is a candidate for scraping. I wasn't previously aware of this issue but we should just move to using the `id` tag in place of `name` going forward.

> if we move to using anchors (as id in header element) that needs new logic here doesn't it?
The scraping code does not care about anchors, right? I guess I don't follow you here.

Reply:
"The scraping code does not care about anchors, right?"  Maybe my language is wrong -- I thought this change was in a section that is just looping over <a> / anchor tags  -- and I don't completely see where the code would build up a link to the new "<h3 id="aimpoint-trending">" location you suggest but heck, it looks like it was just *broken* with what I tried here anyway I and I didn't notice in the test output.